### DISCUSSION TITLE
instance-manager pods are not getting deleted after upgrading longhorn

### DISCUSSION THREAD

**Discussion by vr2388** [longhorn/longhorn#9572] [General]
## Describe the bug

I have installed longhorn 1.6.2 helm chart and updated it to 1.7.0 as i am getting CPU utilisation issue in older version. But after upgrading the helm chart to 1.7.0 then instance-manager pods of older version helm chart are not getting deleted.

## To Reproduce

Install longhorn 1.6.2 helm chart and upgrade it to 1.7.0. After upgrade older instance-manager pods are not getting deleted.

## Expected behavior

Older instance-manager pods should be deleted as it consumes more CPU.

## Environment

 - Longhorn version: 1.6.2-1.7.0
 - Impacted volume (PV): 
 - Installation method (e.g. Rancher Catalog App/Helm/Kubectl): helm
 - Kubernetes distro (e.g. RKE/K3s/EKS/OpenShift) and version:k3s
   - Number of control plane nodes in the cluster:
   - Number of worker nodes in the cluster:
 - Node config
   - OS type and version: Ubuntu 20.04/22.04
   - Kernel version:
   - CPU per node: 
   - Memory per node:
   - Disk type (e.g. SSD/NVMe/HDD):
   - Network bandwidth between the nodes (Gbps):
 - Underlying Infrastructure (e.g. on AWS/GCE, EKS/GKE, VMWare/KVM, Baremetal):
 - Number of Longhorn volumes in the cluster:

**Comment by derekbit** [comment]
You can check https://longhorn.io/kb/troubleshooting-some-old-instance-manager-pods-are-still-running-after-upgrade/.

**Comment by vr2388** [comment]
How can i delete old instance manager pods without detaching the volumes.
